script/rolling-release-scripts/rolling-release-binary-upload.js

```diff
@@ -0,0 +1,69 @@
+// Inspired by Atom's original Nightly Release Script:
+// https://github.com/atom/atom/blob/master/script/vsts/upload-artifacts.js
+
+const fs = require("fs");
+const path = require("path");
+const publish = require("publish-release");
+const packageJson = require("../../package.json");
+
+// Since cirrus always triggers this script, we must check if the version is a rolling
+// release version
+const verSegments = packageJson.version.split(".");
+
+if (verSegments[verSegments.length - 1].length > 4) {
+  console.log(`According to our version: ${packageJson.version} this is not a rolling release...`);
```

Comment:
I think the logic of this check is inverted. Longer than 4 means rolling.

Real-world example: This output seems not accurate: 
> According to our version: 1.108.2023082804 this is not a rolling release...

https://cirrus-ci.com/task/5163254050193408?logs=rolling_upload#L22

Reply:
Oh, whoops, that ones my bad, I'll have to flip that

Reply:
I saw the cron job skipped the real meat of the script due to this, so we can go ahead and get this logic set right and I'll spin up another cron rule so we get another cron build, if that works for you. Thanks for working through all this + review feedback!!

Reply:
I mean, as simple as it is, I think this should do it:

```suggestion
if (verSegments[verSegments.length - 1].length < 4) {
  console.log(`According to our version: ${packageJson.version} this is not a rolling release...`);
```

Reply:
Fixed in last commit